research/synthetic-text-generation-with-DP/requirements.txt

```diff
@@ -0,0 +1,9 @@
+numpy
+pandas==1.5.3
+scikit-learn
+torch==1.12.1
+transformers==4.20.1
+datasets==2.0.0
+prv-accountant==0.1.0
+opacus==1.2.0
+git+https://github.com/microsoft/dp-transformers.git
```

Comment:
Do you have specific version you'd like to use. You can do @<comit_hash> to pin the version. Otherwise future updates might break this.